### PR TITLE
[#208] Rename application class

### PR DESCRIPTION
**Pull request by Tuubz**
#208 

## What happened 👀

Rename application class according to `app-name` argument, there are two processes:
1. Rename application class file (Ex: `CoroutineTemplateApplication` to `MyProjectApplication`)
2. Rename application class in `AndroidManifest.xml`

## Proof Of Work 📹

https://user-images.githubusercontent.com/18277915/169508816-15042a0c-6553-489f-b70c-b262cd2323b0.mov

### CODE REVIEW

File: scripts/new_project.kts [hunk 1]
```diff
@@ -103,6 +107,32 @@ object NewProject {
...
+                    delimiter = fileSeparator,
+                    replacement = "$applicationClassName.kt"
+                )
+                val newApplicationFile = File(newApplicationPath)
+                templateApplicationFile.renameTo(newApplicationFile)
+            }
+
+        File(projectPath)
+            ?.walk()
+            .filter { it.name.endsWith(".kt") || it.name.endsWith(".xml") }
```
Comment (manh-t) [inline scripts/new_project.kts:126]: @Tuubz can we specific `AndroidManifest.xml` file like above? 🙏  is there any other `.kt` files I didn't notice here? 😁

Comment (Tuubz) [reply scripts/new_project.kts:126]: Aaah, good question 👍 

There's `AndroidManifest.xml` but also the application class itself:

Screen Shot 2565-05-20 at 17 39 01

Do you think we should change it? 🙏

**Review by kaungkhantsoe — APPROVED** [review]
LGTM 🚀

File: scripts/new_project.kts (hunk 1 above)
Comment (manh-t) [reply scripts/new_project.kts:126]: ah ha, it's good now. Thanks for explanation 🌮

**Review by manh-t — APPROVED** [review]

File: scripts/new_project.kts (hunk 1 above)
Comment (Wadeewee) [reply scripts/new_project.kts:126]: @Tuubz Which file of `.kt` that we have to rename here 🙏

Comment (Wadeewee) [inline scripts/new_project.kts:126]: @Tuubz Which file of .kt that we have to rename here 🙏

Comment (Tuubz) [reply scripts/new_project.kts:126]: The application class file, ex: `CoroutineTemplateApplication.kt` 🚀

Comment (Wadeewee) [reply scripts/new_project.kts:126]: What about in line `121` 🤔 Is it duplicate 🙏

Comment (Tuubz) [reply scripts/new_project.kts:126]: Oh, I see 👍 

The first part is to rename the file name, but the second part is for inside the file:

Screen Shot 2565-05-20 at 17 39 01

Comment (Wadeewee) [reply scripts/new_project.kts:126]: Oh, I got it now  🚀

Comment (Wadeewee) [reply scripts/new_project.kts:126]: @Tuubz I think we can do like this :

Screen Shot 2565-05-23 at 19 15 41

What do you think? 🙏

**Review by Wadeewee — COMMENTED** [review]
Rest LGTM 🚀

File: scripts/new_project.kts (hunk 1 above)
Comment (Tuubz) [reply scripts/new_project.kts:126]: Sounds good, I'll re-factor 🚀 

I agree, it would be better for easier understanding

**Comment by Tuubz** [comment]
@Wadeewee Updated 🚀

**Review by Wadeewee — APPROVED** [review]

**Pull request merged**